integration/specs/DateTimePicker/datetimepicker-1.spec.js

```diff
@@ -0,0 +1,22 @@
+const PageDateTimePicker = require('../../../src/components/DateTimePicker/pageObject');
+
+const DATETIMEPICKER = '#datetimepicker-1';
+
+describe('DateTimePicker', () => {
+    beforeAll(() => {
+        browser.url('/#!/DateTimePicker/1');
+    });
+    beforeEach(() => {
+        browser.refresh();
+        const component = $(DATETIMEPICKER);
+        component.waitForExist();
+    });
+    it('should set correct date on modal header when day is clicked', () => {
+        const picker = new PageDateTimePicker(DATETIMEPICKER);
+        picker.click();
+        picker.waitUntilModalIsOpen();
+        expect(picker.getModalHeaderText()).toBe('10/25/19, 10:44 AM');
+        picker.clickDay(10);
+        expect(picker.getModalHeaderText()).toBe('10/10/19, 10:44 AM');
```

Comment:
Remove this test and the getModalHeaderText method in the page object since the time is not changed we don't know if we will maintain this.
Then add a test for:
```should set the right value when select a date```
create a method getValue in the page object that will return the input value so you can check that the right date time is selected, for now we will not implement a method for set the time, so left it as 12:00